test/dynamo/test_ctx_manager.py

```diff
@@ -287,6 +287,24 @@ def fn(x):
         self.assertEqual(cnts.frame_count, 1)
         self.assertEqual(cnts.op_count, 21)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "requires cuda")
+    def test_cuda_stream_is_not_none(self):
+        def fn(x):
+            x = torch.mul(x, 1)
+            x = torch.add(x, 2)
+
+            cur_stream = torch.cuda.current_stream()
+            if cur_stream is not None:
```

Comment:
This tests for `operator.is_` `is`, the additional handling is for `operator.eq` `==` though? So do we actually have coverage here?

Reply:
Before the PR,  we've supported operator.eq stream and stream or event and event. 

This PR adds support for comparing stream and constant. I feel in this case, the comparison logic is the same for built in comparison operators, e.g. is, is_not_, etc.

Reply:
Added a test for `!= "const_str"`.